### PR TITLE
Fixing signs permission cancelling, add new messages & sound

### PR DESCRIPTION
**Pull request by montlikadani**
All the messages in the _creator_ folder can not do it because it's not so easy.

### CODE REVIEW

File: src/pl/plajer/pinata/SignManager.java [hunk 1]
```diff
@@ -58,11 +62,11 @@ public void onSignDestroy(BlockBreakEvent e) {
...
             if(s.getLine(0).equals(Utils.colorMessage("Signs.Lines.First"))) {
-                if(e.getPlayer().hasPermission("pinata.admin.sign.destroy")) {
-                    //todo wtf
-                    e.getPlayer().sendMessage(Utils.colorMessage("Signs.Invalid-Pinata"));
-                } else {
+                if(!e.getPlayer().hasPermission("pinata.admin.sign.destroy")) {
                     e.getPlayer().sendMessage(Utils.colorMessage("Signs.No-Permission"));
+                    e.setCancelled(true);
+					return;
+                } else {
```
Comment (Plajer) [inline src/pl/plajer/pinata/SignManager.java]: ???

**Comment by Plajer** [comment]
Thanks for saving my time 🥇

**Comment by Plajer** [comment]
lol

**Comment by montlikadani** [comment]
Then wut??

File: resources/messages.yml [hunk 2]
```diff
@@ -115,4 +125,4 @@ Other:
   Plugin-Up-To-Date: "&e[Pinata] There is plugin update! Your version: %old%, new version: %new%"
   Plugin-Update-Check-Failed: "&c[Pinata] Failed to check for update: %error%"
 
-File-Version-Do-Not-Edit: 10
\ No newline at end of file
+File-Version-Do-Not-Edit: 10
```
Comment (montlikadani) [inline resources/messages.yml:128]: Ooops.. Version up...

**Pull request merged**